templates/controllers/modals/submissionMetadata/form/issueEntrySubmissionReviewForm.tpl

```diff
@@ -17,6 +17,7 @@
 		$('#{$submissionMetadataViewFormId}').pkpHandler(
 			'$.pkp.controllers.form.FileUploadFormHandler',
 			{ldelim}
+				$readOnly: {$formParams.readOnly|escape},
```

Comment:
`$readonly` should be `readonly`. In JS-land, we use the `$` prefix on variable names only when we're referencing a jQuery object. So `$variable = $(selector)` but `variable = true'`.

That gets especially confusing when writing JavaScript in PHP templates, but it's a useful distinction in the JS-world. :)